applications/nrf5340_audio/src/bluetooth/bt_ctrl_cfg.c

```diff
@@ -0,0 +1,203 @@
+
+#define WDT_TIMEOUT_MS	      1200
+#define CTRL_POLL_INTERVAL_MS (WDT_TIMEOUT_MS - 200)
+
+static struct k_work work_ctrl_version_poll;
+static void ctlr_version_poll_timer_handler(struct k_timer *timer_id);
+static int wdt_ch_id;
+
+K_TIMER_DEFINE(ctlr_poll_timer, ctlr_version_poll_timer_handler, NULL);
+
+static int ctrl_acs_nrf53_cfg(void)
+{
+	int ret;
+	/* Enable notification of lost ISO packets */
+	ret = ble_hci_vsc_op_flag_set(BLE_HCI_VSC_OP_ISO_LOST_NOTIFY, 1);
+	if (ret) {
+		return ret;
+	}
+
+#if (CONFIG_NRF_21540_ACTIVE)
+	/* Indexes for the pins gotten from nrf21540_ek_fwd.overlay */
+	uint8_t tx_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						     gpios, 0);
+	uint8_t rx_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						     gpios, 1);
+	uint8_t pdn_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						      gpios, 2);
+	uint8_t ant_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						      gpios, 3);
+	uint8_t mode_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(
+		DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if), gpios, 4);
+
+	struct ble_hci_vs_cp_nrf21540_pins nrf21540_pins = {
+		.mode = mode_pin,
+		.txen = tx_pin,
+		.rxen = rx_pin,
+		.antsel = ant_pin,
+		.pdn = pdn_pin,
+		/* Set CS pin to ffff since we are not using the SPI */
+		.csn = 0xffff};
+
+	ret = ble_hci_vsc_nrf21540_pins_set(&nrf21540_pins);
+	if (ret) {
+		return ret;
+	}
+
+	ret = ble_hci_vsc_radio_high_pwr_mode_set(
+		MAX(CONFIG_NRF_21540_MAIN_DBM, CONFIG_NRF_21540_PRI_ADV_DBM));
+	if (ret) {
+		return ret;
+	}
+
+	ret = ble_hci_vsc_adv_tx_pwr_set(CONFIG_NRF_21540_MAIN_DBM);
+	if (ret) {
+		return ret;
+	}
+
+	LOG_DBG("TX power set to %d", CONFIG_NRF_21540_MAIN_DBM);
+
+	ret = ble_hci_vsc_pri_adv_chan_max_tx_pwr_set(CONFIG_NRF_21540_PRI_ADV_DBM);
+	if (ret) {
+		return ret;
+	}
+
+	LOG_DBG("Primary advertising TX power set to %d", CONFIG_NRF_21540_PRI_ADV_DBM);
+#else
+	ret = ble_hci_vsc_adv_tx_pwr_set(CONFIG_BLE_ADV_TX_POWER_DBM);
+	if (ret) {
+		return ret;
+	}
+
+	LOG_DBG("TX power set to %d", CONFIG_BLE_ADV_TX_POWER_DBM);
+
+	/* Disabled by default, only used if another TX power for primary adv channels is needed */
+	ret = ble_hci_vsc_pri_adv_chan_max_tx_pwr_set(BLE_HCI_VSC_PRI_EXT_ADV_MAX_TX_PWR_DISABLE);
+	if (ret) {
+		return ret;
+	}
+
+#endif /*CONFIG_NRF_21540_ACTIVE*/
+
+	/* Map controller LEDs*/
+	ret = ble_hci_vsc_led_pin_map(PAL_LED_ID_CPU_ACTIVE,
+				      DT_GPIO_FLAGS_BY_IDX(DT_NODELABEL(rgb2_green), gpios, 0),
+				      DT_GPIO_PIN_BY_IDX(DT_NODELABEL(rgb2_green), gpios, 0));
+	if (ret) {
+		return ret;
+	}
+
+	ret = ble_hci_vsc_led_pin_map(PAL_LED_ID_ERROR,
+				      DT_GPIO_FLAGS_BY_IDX(DT_NODELABEL(rgb2_red), gpios, 0),
+				      DT_GPIO_PIN_BY_IDX(DT_NODELABEL(rgb2_red), gpios, 0));
+	if (ret) {
+		return ret;
+	}
+
+	return 0;
+}
+
+int bt_ctrl_version_get(uint16_t *ctrl_version)
```

Comment:
Move this below static functions, it's already defined in the .h file, so no need to have it above